neon_utils/hana_utils.py

```diff
@@ -126,6 +146,15 @@ def request_backend(endpoint: str, request_data: dict,
     @param server_url: Base URL of Hana server to query
     @returns: dict response
     """
+    global _client_config
+    global _headers
+    if not server_url:
+        set_default_backend_url()
+        server_url = _DEFAULT_BACKEND_URL
+    if server_url != _DEFAULT_BACKEND_URL and _client_config:
+        LOG.warning(f"Using new remote: {server_url}")
```

Comment:
Reviewing on mobile so possibly missing some context, but why warning?

Reply:
Good point; this is really just `INFO`